### PR TITLE
FOGL-3083: Update to Angular v8

### PR DESCRIPTION
**Pull request by shariq1501**

### CODE REVIEW

**Review by MonikaSharma06 — COMMENTED** [review]
Please update `developers-guide` also.

File: .browserslistrc [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Browsers that we support
+
+last 2 Chrome versions
+last 2 ChromeAndroid versions
+
+Safari >= 11
+
+last 2 Firefox versions
+last 1 FirefoxAndroid versions
```
Comment (praveen-garg) [inline .browserslistrc]: Remove please.

File: .browserslistrc [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Browsers that we support
+
+last 2 Chrome versions
+last 2 ChromeAndroid versions
+
+Safari >= 11
```
Comment (praveen-garg) [inline .browserslistrc:6]: What about iOS?

File: .browserslistrc [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Browsers that we support
```
Comment (praveen-garg) [inline .browserslistrc:1]: should we remove browsers targets from .babelrc ?

File: src/polyfills.ts [hunk 4]
```diff
@@ -38,8 +38,8 @@
 // import 'classlist.js';  // Run `npm install --save classlist.js`.
 
 /** Evergreen browsers require these. **/
-import 'core-js/es6/reflect';
-import 'core-js/es7/reflect';
+// import 'core-js/es6/reflect';
+// import 'core-js/es7/reflect';
```
Comment (praveen-garg) [inline src/polyfills.ts:42]: why these have been commented. should we remove?

**Review by praveen-garg — CHANGES_REQUESTED** [review]

**Comment by shariq1501** [comment]
1. README and developers-guide 
   >Done

2. e2e 
   >Working
3. build size info current develop vs this
   > No change `INFO:  Deployable dist size   6.7M	dist`  on both branch
4. change-logs if any needed 
   > Updated

5. Removed `rxjs-compat` and `babel` dependencies 

6. App will now be more performant, thanks to differential loading. two bundles are created when building for production: a bundle for modern browsers that support ES6  and a bundle for older browsers that only support the ES5 version of JavaScript. **We are creating only one build as we are supporting ES6 compatible browsers only**

7. Import syntax changed for lazy loading routes. Those are more dynamic now like 
`{
    path: 'south', loadChildren: () => import('./components/core/south/south.module')
      .then(m => m.SouthModule)
  }`

**Review by praveen-garg — DISMISSED** [review]

**Review by MonikaSharma06 — DISMISSED** [review]

File: docs/changelog.md [hunk 5]
```diff
@@ -1,3 +1,8 @@
+## v1.x.x [xxxx-xx-xx]
```
Comment (praveen-garg) [inline docs/changelog.md]: v1.7.0next [2019-xx-xx]

File: docs/changelog.md [hunk 6]
```diff
@@ -1,3 +1,8 @@
...
+
+#### Changed
+* Upgraded to Angular 8 (including compatible TypeScript / CLI versions)
+
```
Comment (praveen-garg) [inline docs/changelog.md:5]: also add, 
`Fixed notification service install and enabled check to get latest state`

>under Fixed

**Review by praveen-garg — APPROVED** [review]

**Pull request merged**